### PR TITLE
fix: proposal glitches/errors

### LINKED ISSUE #255: Proposal detail page glitches/errors
**Describe the bug**
When being on the proposal detail page several issues occur which should not (date/time, voting period not respected).

**To Reproduce**
1. Create a proposal with little text and start date in the future
2. go to the proposal detail page
3. see that the start time is missing in the right info box
4. see that the load more button is visible even though there is no more content
5. see that you can already cast your vote 
6. cast your vote before the start time 
7. see that it goes through
8. wait until the voting time is over
9. go to the proposal detail page
10. see that the status is expired but you can still cast your vote

**Expected behavior**
1. The voting can only take place during voting period, not before and not after the voting period. 
2. The voting period is visible with a timezone indication: hh:mm UTC
3. the load more button appears only when there is more content than it fits in to the visible content box.

**Screenshots**
image

### PR DESCRIPTION
**Pull request by hadi-ibra**
### closes #255 
-------------

### Done:

- Start time is now the current time the user is creating the proposal and not cached
- Voting can be only done during the voting period
- Show more button is hidden when the description is less than 250 characters
- Description is truncated when more than 250 characters
- When clicking on show more button it displays the rest of the description and changes to show less
- When show less is clicked it truncates the text and changes to show more

### CODE REVIEW

**Comment by 5-mark** [comment]
@hadi-ibra preview does not load the page. cannto review.

**Comment by hadi-ibra** [comment]
> @hadi-ibra preview does not load the page. cannot review.

This is because of the chain upgrade

**Comment by 5-mark** [comment]
@hadi-ibra Cannot create proposal. Configured multiple different start & end dates in the future but always get this error: 

image


image

**Comment by hadi-ibra** [comment]
@5-mark yes I'll check it out

**Comment by 5-mark** [comment]
@hadi-ibra 

❌  Start time is now the current time the user is creating the proposal and not cached 
--> in the meantime proposal duration is 5 minutes minimum 
❌  Voting can be only done during the voting period
--> i still can vote, not even being a member of the dao --> chain throws error but frontend shouldnt allow it. 
❌ Show more button is hidden when the description is less than 250 characters
--> is this really the "fix"
image
huge whitespaces (blackspaces) on the proposal.

❓ Description is truncated when more than 250 characters
--> yes it is truncated, but why at 250 characters. See picture above.

✅  When clicking on show more button it displays the rest of the description and changes to show less
✅  When show less is clicked it truncates the text and changes to show more

**Comment by hadi-ibra** [comment]
@5-mark the ones you checked wrong were working on my side I will check them, since this pull request is a little bit old it needs some updates

**Comment by 5-mark** [comment]
Steps to reproduce the behavior:
@hadi-ibra 
1. Create a proposal with little text and start date in the future
2. go to the proposal detail page
3. see that the start time is missing in the right info box ❌ 
image

4. see that the load more button is visible even though there is no more content ✅ 
5. see that you can already cast your vote ❌ 
6. cast your vote before the start time
7. see that it goes through ❌ 
 

9. wait until the voting time is over
10. go to the proposal detail page
11. see that the status is expired but you can still cast your vote ✅

**Comment by 5-mark** [comment]
@hadi-ibra whats the status here? PR is open for 15days now. Did you consider my last review?

File: pkg/app/src/components/TabPanels/Proposal/detail.tsx [hunk 1]
```diff
@@ -188,10 +205,16 @@ export function ProposalDetail({ proposalId, goBack }: ComponentProps) {
 
 						{matches && <Chip color="secondary" label={proposalTypeName} variant="outlined" />}
 					</Stack>
-					<Typography variant="body1">{proposal.proposal_metadata?.description ?? ''}</Typography>
-					<Button sx={{ alignSelf: 'center' }} size="large" variant="outlined">
-						{t('button:ui:show_more')}
-					</Button>
+					<Typography variant="body1">
+						{showMore
```
Comment (developer-sidani) [inline pkg/app/src/components/TabPanels/Proposal/detail.tsx:209]: @hadi-ibra useMemo

Comment (hadi-ibra) [reply pkg/app/src/components/TabPanels/Proposal/detail.tsx]: done

**Review by developer-sidani — APPROVED** [review]
LGTM!

**Pull request merged**